DOCS-2735: update manual payout steps (#1650)
MultiSafepay/docs: content/account/manage-your-balance/making-payouts.md

To make manual payouts, follow these steps:

1. Sign in to your [MultiSafepay account](https://merchant.multisafepay.com).
2. Go to **Finance** > **Payouts**.

> 3. Under **Withdraw funds**, select the currency, enter an amount, and click **Withdraw funds**.

Reviewer: ```suggestion
3. Under **Withdraw funds**:  
    - Select the currency.
    - Enter an amount.
    - Click **Withdraw funds**.
```

> 4. Check the details of your withdrawal, click **Submit**.

Reviewer: ```suggestion
4. Check the details of the payout, and then click **Submit**.
```
